lawnchair/src/app/lawnchair/LawnchairApp.kt

```diff
@@ -201,6 +203,9 @@ class LawnchairApp : Application() {
         @JvmStatic
         val isRecentsEnabled: Boolean get() = instance.recentsEnabled
 
+        @JvmStatic
+        val isAtleastT: Boolean get() = instance.isAtleastT
```

Comment:
Why do we need this when we already have `Utilities.ATLEAST_T`?

Reply:
So we know that we're the one who applied when merging A14 so it will easy to fixing conflict